### ISSUE TITLE
instruction for switching

### ISSUE THREAD

**Issue by paxarpp** [lttb/reshadow#20]
Good day,
Is there a described instruction for switching an application from styled-component to reshadow?

**Comment by lttb** [comment]
Hi @paxarpp,

Thank you for the interest!

I'm working on updated documentation including switching from `styled-components` and other examples, hope to release it in coming days.

But I think that the basic transition should be easy at all, there are a few steps:
1) `npm i -S @reshadow/styled`
2) replace imports from `styled-components` to `@reshadow/styled`

The one thing, that is important to consider, if you have some components like this (styling a custom React Component with dynamic values), please make sure that the decorated component passes the `style` prop to the DOM as well as the `className`.

```
const Button = ({className, style, children}) => (
  <button className={className} style={style}>{children}</button>
)

const MyButton = styled(Button)`
  color: ${props => props.color};
`
```

There are also some examples:
- switch for ShizukuIchi/winXP
  https://github.com/lttb/winXP/commit/dcc79422bb85b6f140c40ec9b43b9b42c3ec166a
- example with `styled-system`
  https://codesandbox.io/s/styledsystemreshadowstyled-3yeho
- example with benchmarks
  https://codesandbox.io/embed/cssinjsbenchmarks-xggtc?fontsize=14&module=%2Fsrc%2Fbenchmarks%2Freshadow-styled.js

So would be glad for any feedback, especially if we have missed some cases 🙂

**Comment by paxarpp** [comment]
![after](https://user-images.githubusercontent.com/34523588/58761955-9d09fa00-8552-11e9-9f6b-a8972d8c74b9.JPG)

Thanks, the transition is really simple, but in the Storybook my components have an artifact (this is the pseudo-element :after, but it is not in the styles of my component)

**Comment by lttb** [comment]
@paxarpp thank you for the issue!

could you provide a link to the repo or something like minimal example please?
https://codesandbox.io/s/reshadowstyled-dg83s

**Comment by paxarpp** [comment]
https://github.com/paxarpp/ui_library/tree/future/reshadow

**Comment by lttb** [comment]
> https://github.com/paxarpp/ui_library/tree/future/reshadow

Thank you very much!

I think I found a bug, try to fix it today 😄

**Comment by lttb** [comment]
Hey @paxarpp, I've just published `@reshadow/styled@0.0.1-alpha.52` where this problem looks fixed
https://github.com/lttb/ui_library/commit/5b1f1cc1dea930cf96a673fc3681522b3a9b5b6b

**Comment by paxarpp** [comment]
ok, try now
